error on schema update

the python library released fine, but not the schema.

https://github.com/dandi/dandischema/runs/2779002429?check_suite_focus=true

i updated the schema repo manually.

```
releases/0.4.1/published-dandiset.json
remote: Permission to dandi/schema.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/dandi/schema/': The requested URL returned error: 403
Error: Process completed with exit code 128.
```
so smells like wrong it not good enough GITHUB_TOKEN was used , right @jwodder ?

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -77,6 +77,7 @@ jobs:
         with:
           repository: dandi/schema
           path: schema
+          token: ${{ secrets.DANDI_GITHUB_TOKEN }}
 
       - name: Test for unversioned changes
         run: |

